app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,27 @@
 class Api::BaseController < ApplicationController
   layout nil
-end
+
+  # POST /api/validate_email
+  def validate_email
+    email = params[:email]
+    if email_valid?(email)
+      if User.email_exists?(email)
+        AuthenticationEvent.new.log_email_validation(email, 'email_validation', current_user&.id)
+        render json: { accountExists: true, nextStep: 'authentication' }
+      else
+        render json: { accountExists: false, error: I18n.t('errors.messages.email_not_found') }, status: :not_found
+      end
+    else
+      render json: { error: I18n.t('errors.messages.invalid_email_format') }, status: :unprocessable_entity
```

Comment:
The `email_valid?` method should be moved to a validator or a service object to keep the controller thin and adhere to the single responsibility principle.